src/topology/metric_space/basic.lean

```diff
@@ -2107,6 +2107,13 @@ end
 @[simp] lemma bounded_empty : bounded (∅ : set α) :=
 ⟨0, by simp⟩
 
+lemma nonempty_of_unbounded (h : ¬ bounded s) : s.nonempty :=
```

Comment:
Could you please cherry-pick changes to this file and `emetric_space`, and merge them?

Reply:
What would you want the split to be?

Reply:
Given that the freeze is in two days, I would rather have it all at once. The PR is quite small already.